core/test/utils/fixtures/export/export-basic-test.json

```diff
@@ -1197,6 +1197,15 @@
                         "created_by": "1",
                         "updated_at": "2017-09-01T12:29:50.000Z",
                         "updated_by": "1"
+                    },
+                    {
+                        "id": "59a952be7d79ed06b0d21137",
+                        "name": "Contributor",
+                        "description": "Contributors",
+                        "created_at": "2017-09-01T12:29:50.000Z",
+                        "created_by": "1",
+                        "updated_at": "2017-09-01T12:29:50.000Z",
+                        "updated_by": "1"
```

Comment:
You've added the contributor to the 1.x export example, but i can't find the assertion? 🤔